packages/amplication-server/src/schema.graphql

```diff
@@ -1504,7 +1505,7 @@ input CreateGitRepositoryInput {
   gitOrganizationId: String!
   gitProvider: EnumGitProvider!
   gitOrganizationType: EnumGitOrganizationType!
-  resourceId: String!
+  resourceId: String
```

Comment:
Can this be really optional? What happen if it is not passed in the mutation data?

Reply:
Yes. This is a part of the refactoring we did in the git provider section. In the service wizard you don't have a resource yet, as you are creating the service